docs/admin/federation/index.md

```diff
@@ -0,0 +1,103 @@
+---
+---
+This guide explains how to set up a federation cluster. Federation is an alpha feature which lets you control multiple kubernetes clusters.
+[Federation proposal](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/federation.md) has more details.
+
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+This guide assumes that you have a running kubernetes cluster.
+If not, then you can head over to the [getting started guides](/docs/getting-started-guides/) to bring up a cluster.
+
+This guide also assumes that you have the kubernetes source code.
+If not, then you can head over to the [download kubernetes guide](/docs/getting-started-guides/binary_release/) to get the source code.
+
+## Setting up federation control plane
+
+### Building and pushing federation control binaries
+
+You can build the federation control plane binaries using the following command:
```

Comment:
Yes. We do not do it right now.
But its a reasonable thing to expect.
Filed https://github.com/kubernetes/kubernetes/issues/27382